distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numeric/Rounding_Mode.enso

```diff
@@ -0,0 +1,19 @@
+polyglot java import java.math.RoundingMode
+
+## A wrapper around the Java
+   [`RoundingMode`](https://docs.oracle.com/javase/8/docs/api/java/math/RoundingMode.html)
+    class.
+
+   `Rounding_Mode` is used to specify the method to use for rounding a value in
+   operations on `Decimal`s.` See `Decimal` for more details.
+type Rounding_Mode
+    Value (rounding_mode:RoundingMode)
+
+    # Round towards positive infinity. (This is the default.)
+    half_up : Rounding_Mode
+    half_up = Rounding_Mode.Value RoundingMode.HALF_UP
+
+    ## Rownd towards the nearest neighbor, with ties broken by rounding towards
+       the nearest even neighbor.
+    bankers : Rounding_Mode
+    bankers = Rounding_Mode.Value RoundingMode.HALF_EVEN
```

Comment:
Should we start getting this `Rounding_Mode` in `Number.round` as well instead of `use_bankers:Boolean`? I think that would be clearer. There were already discussions that we want to use less Boolean flags because their meaning is not as clear in the GUI.

Ofc fine (and preferred) to do as a separate PR/ticket, as that is not related to Decimal.

Reply:
#9387